src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
```

Comment:
**Reporter**: FINDBUGS
**Rule**: EQ_ALWAYS_TRUE
**Severity**: ERROR
**File**: se/bjurr/violations/lib/example/MyClass.java L35
**Source**: se.bjurr.violations.lib.example.MyClass

equals method always returns true


  <p> This class defines an equals method that always returns true. This is imaginative, but not very smart.
Plus, it means that the equals method is not symmetric.
</p>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*